pydantic_ai_slim/pydantic_ai/models/__init__.py

```diff
@@ -611,6 +613,79 @@ def _cached_async_http_transport() -> httpx.AsyncHTTPTransport:
     return httpx.AsyncHTTPTransport()
 
 
+@overload
+async def download_item(
+    item: FileUrl,
+    data_format: Literal['bytes'],
+    type_format: Literal['mime', 'extension'] = 'mime',
+) -> tuple[bytes, str]:
+    pass
+
+
+@overload
+async def download_item(
+    item: FileUrl,
+    data_format: Literal['base64', 'base64_uri', 'text'],
+    type_format: Literal['mime', 'extension'] = 'mime',
+) -> tuple[str, str]:
```

Comment:
Instead of `tuple[str, str]`, can we have a `DownloadedItem` `TypedDict` instead?